### PR TITLE
Page for deployment

### PR DESCRIPTION
**Pull request by vidmantas**
Updated version, for discussion where more info is needed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,14 +19,39 @@ <h1 class="logo">
...
+
+          <div class="talk">
+            <h3 class="title">More to be announced soon!</h3>
+          </div>
+
+          <h2>Call for papers</h2>
+
+          <div class="cfp">
+            <p>Would love to give a talk at RubyConfLT'15?</p>
+            <p>Please <a href="#">register</a> until 2015-02-28, and we will get in touch with you</p>
```
Comment (vidmantas) [inline index.html]: Let's copy CFP form from Euruko?

File: index.html [hunk 2]
```diff
@@ -74,14 +100,19 @@ <h3 class="title">Party</h3>
               <a href="http://www.vinted.com"><img src="assets/vinted.png" /></a>
             </li>
             <li>
-              <a href="http://www.samesystem.com"><img src="assets/samesystem.png" /></a>
-            </li>
-            <li>
               <a href="http://www.necolt.com"><img src="assets/necolt.png" /></a>
             </li>
+            <!-- ADD HERE: Synergium, DaWanda & Wise Monks -->
```
Comment (vidmantas) [inline index.html]: Who has them?

**Comment by vidmantas** [comment]
Also: we need a link to subscribe to newsletters

**Comment by vidmantas** [comment]
@arturaspiksrys gal galėtum savo logo įdėti?

File: index.html (hunk 1 above)
Comment (giedriusr) [inline index.html]: Yes, why not.

**Comment by giedriusr** [comment]
Who will be dealing with newsletters?

**Pull request merged**